.evergreen/config.yml

```diff
@@ -910,10 +916,12 @@ buildvariants:
   tasks:
     - .serverless
 
-# Load balancer is available from MongoDB 5.0+
 - matrix_name: "test-loadBalanced"
-  matrix_spec: { "os": "debian11", "mongodb-versions": ["5.0", "6.0"], "php-edge-versions": "latest-stable", "driver-versions": "latest-stable" }
+  matrix_spec: { "os": "debian11", "mongodb-versions": "*", "php-edge-versions": "latest-stable", "driver-versions": "latest-stable" }
   display_name: "Load balanced - ${mongodb-versions}"
+  exclude_spec:
+    # Load balancer is available from MongoDB 5.0+
+    - { "os": "debian11", "mongodb-versions": ["3.6", "4.0", "4.2", "4.4"], "php-edge-versions": "latest-stable", "driver-versions": "latest-stable" }
```

Comment:
The `test-csfle-skip_crypt_shared` and `test-csfle-without_aws_creds` matrices below currently use 6.0 exclusively. Let me know if you think it makes sense to test on 6.0+ using an `exclude_spec` similar to what we have here.

Reply:
With 7.0 coming soon, it would make sense to test on 6.0+ and use an `exclude_spec` for anything that isn't supported.